src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1183,6 +1205,112 @@ protected static void VerifyFlowGraphForTest<TSyntaxNode>(CSharpCompilation comp
+                case OperationKind.Stop:
+                case OperationKind.End:
+                case OperationKind.RaiseEvent:
+                case OperationKind.Literal:
+                case OperationKind.Conversion:
+                case OperationKind.Invocation:
+                case OperationKind.ArrayElementReference:
+                case OperationKind.LocalReference:
+                case OperationKind.ParameterReference:
+                case OperationKind.FieldReference:
+                case OperationKind.MethodReference:
+                case OperationKind.PropertyReference:
+                case OperationKind.EventReference:
+                case OperationKind.AnonymousFunction:
+                case OperationKind.ObjectCreation:
+                case OperationKind.TypeParameterObjectCreation:
+                case OperationKind.ArrayCreation:
+                case OperationKind.InstanceReference:
+                case OperationKind.IsType:
+                case OperationKind.Await:
+                case OperationKind.SimpleAssignment:
+                case OperationKind.CompoundAssignment:
+                case OperationKind.Parenthesized:
+                case OperationKind.EventAssignment:
+                case OperationKind.InterpolatedString:
+                case OperationKind.AnonymousObjectCreation:
+                case OperationKind.NameOf:
+                case OperationKind.Tuple:
+                case OperationKind.DynamicObjectCreation:
+                case OperationKind.DynamicMemberReference:
+                case OperationKind.DynamicInvocation:
+                case OperationKind.DynamicIndexerAccess:
+                case OperationKind.TranslatedQuery:
+                case OperationKind.DelegateCreation:
+                case OperationKind.DefaultValue:
+                case OperationKind.TypeOf:
+                case OperationKind.SizeOf:
+                case OperationKind.AddressOf:
+                case OperationKind.IsPattern:
+                case OperationKind.Increment:
+                case OperationKind.Throw:
+                case OperationKind.Decrement:
+                case OperationKind.DeconstructionAssignment:
+                case OperationKind.DeclarationExpression:
+                case OperationKind.OmittedArgument:
+                case OperationKind.VariableDeclarator:
+                case OperationKind.VariableDeclaration:
+                case OperationKind.Argument:
+                case OperationKind.InterpolatedStringText:
+                case OperationKind.Interpolation:
+                case OperationKind.ConstantPattern:
+                case OperationKind.DeclarationPattern:
+                case OperationKind.UnaryOperator:
+                case OperationKind.FlowCapture:
+                case OperationKind.FlowCaptureReference:
+                    return true;
+            }
+
+            Assert.True(false, $"Unhandled node kind OperationKind.{n.Kind}");
```

Comment:
Given this is test code, it seems we can throw `UnexpectedValue` to fail both debug and release tests.

Reply:
"Assert.True" is not a Debug.Assert

---
In reply to: [163071284](https://github.com/dotnet/roslyn/pull/24358#discussion_r163071284) [](ancestors = 163071284)